NetCasbin/CoreEnforcer.cs

```diff
@@ -382,7 +432,30 @@ public Task<bool> EnforceAsync(params object[] requestValues)
         /// <returns>Whether to allow the request.</returns>
         public bool Enforce(params object[] requestValues)
         {
-            return Enforce((IReadOnlyList<object>) requestValues);
+#if !NET45
```

Comment:
Duplicated code with `EnforceAsync` above.

Reply:
These codes are different.

Reply:
Ok, found it, `TryGetResultAsync` and `TryGetResult`